tests/phpunit/class-wc-stripe-payment-gateway-test.php

```
@@ -1715,4 +1715,41 @@ public function provide_test_generate_payment_request_shipping_phone_cases(): ar
+		$output = ob_get_clean();
+
+		$this->assertStringNotContainsString( '<fieldset', $output );
+		$this->assertMatchesRegularExpression( '/^\s*<div\b/', $output );
+		$this->assertStringContainsString( 'woocommerce-SavedPaymentMethods-saveNew', $output );
+
+		if ( $force_checked ) {
+			$this->assertStringContainsString( '<div style="display:none;">', $output );
+			$this->assertStringContainsString( 'checked', $output );
+		} else {
+			$this->assertStringNotContainsString( '<div style="display:none;">', $output );
+		}
```

PHPMD (2.15.0)

[error] The method test_save_payment_method_checkbox_does_not_render_fieldset_wrapper uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)